package.json

```diff
@@ -5,10 +5,10 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build:be": "./node_modules/typescript/bin/tsc -p ./src/backend",
+    "build:be": "node ./node_modules/typescript/bin/tsc -p ./src/backend",
```

Comment:
tsc is a binary. Why use node to run it?

Reply:
Not sure if it's a windows issue (I'm on windows 10) but it it was saying - "." not recognized as a command. Adding "node" fixed this.

Reply:
Works for me too.